examples/image_pixmap_chessboard.cpp

```diff
@@ -0,0 +1,70 @@
+/*=============================================================================
+   Copyright (c) 2016-2020 Joel de Guzman
+
+   Distributed under the MIT License (https://opensource.org/licenses/MIT)
+=============================================================================*/
+#include "app.hpp"
+
+using namespace cycfi::artist;
+
+bool constexpr is_little_endian() {
```

Comment:
Use infra/support.hpp: https://github.com/cycfi/infra/blob/master/include/infra/support.hpp#L356 (BTW, is_little_endian can't be constexpr bacuse it needs to run on the actual machine to make a determination)

Reply:
Cool! Missed that. Will update!

Thanks for the info :)